### PR TITLE
Update eks-blueprints.tf terraform version of VPC module

### PR DESCRIPTION
**Pull request by mseverin**
Updated terraform-aws-module-vpc to >4.0

*Issue #, if available:*

*Description of changes:*
Updated the version of the terraform-aws-module-vpc to get past errors when running the lab

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/using_ec2_spot_instances_with_eks/010_prerequisites/prerequisites.files/eks-blueprints.tf [hunk 1]
```diff
@@ -159,7 +159,7 @@ module "eks_blueprints_kubernetes_addons" {
 #---------------------------------------------------------------
 module "vpc" {
   source  = "terraform-aws-modules/vpc/aws"
-  version = "~> 3.0"
+  version = "~> 4.0"
```
Comment (chrismld) [inline content/using_ec2_spot_instances_with_eks/010_prerequisites/prerequisites.files/eks-blueprints.tf:162]: Thanks @mseverin for your contribution! Would you mind changing this to a version like `4.0.2` instead? Also, please update your branch from the main one as we recently pushed a new change there. Thanks.

**Review by chrismld — CHANGES_REQUESTED** [review]
Thanks for your contribution @mseverin :)

**Comment by mseverin** [comment]
Updated to recommend version

File: content/using_ec2_spot_instances_with_eks/010_prerequisites/prerequisites.files/eks-blueprints.tf (hunk 1 above)
Comment (mseverin) [reply content/using_ec2_spot_instances_with_eks/010_prerequisites/prerequisites.files/eks-blueprints.tf]: Done

**Review by chrismld — APPROVED** [review]

**Pull request merged**